v3/src/components/graph/adornments/plotted-value/plotted-value-model.ts

```diff
@@ -14,19 +16,25 @@ export const PlottedValueModel = AdornmentModel
       // This is just a proof-of-concept placeholder for function strings. It will always return
       // the mean of caseValues no matter what the string value is.
       return mean(caseValues)
+    },
+    getCaseValues(attrId: string, cellKey: Record<string, string>, dataConfig: IDataConfigurationModel) {
```

Comment:
Just to note: I added a [very similar view to `UnivariateMeasureAdornmentModel`](https://github.com/concord-consortium/codap/blob/5afdf315fc5db55178261fff23c125f4b02b7a40/v3/src/components/graph/adornments/univariate-measures/univariate-measure-adornment-model.ts#L32-L42) in the the Mean and Median adornments PR. I _think_ it'd make sense to move this to the main `AdornmentModel` and just have a single instance of the view there, but I'm going to try doing that as part of the work on the Standard Deviation adornment.